taskrunner/host_gateway.go

```diff
@@ -229,10 +234,18 @@ func startAgentOnRemote(settings *evergreen.Settings, hostObj *host.Host, sshOpt
 		fmt.Sprintf("--log_prefix='%s'", filepath.Join(hostObj.Distro.WorkDir, agentFile)),
 		fmt.Sprintf("--working_directory='%s'", hostObj.Distro.WorkDir),
 	}
+	if hostObj.Distro.SetupAsSudo {
+		agentCmdParts = append(agentCmdParts, "--setup_as_sudo")
+	}
 
 	// build the command to run on the remote machine
 	remoteCmd := strings.Join(agentCmdParts, " ")
-	grip.Info(remoteCmd)
+	m := message.Fields{
+		"message": "running remote script on agent",
+		"host":    hostObj.Id,
+		"command": remoteCmd,
+	}
+	grip.Info(m)
```

Comment:
nit: just pass the message literal to the grip call.

less nit: put the command id from line 259 (new) in this message to make the logs easier to correlate.

Reply:
done